kafka/src/main/java/brave/example/ProducerService.java

```diff
@@ -0,0 +1,31 @@
+package brave.example;
+
+import brave.kafka.clients.KafkaTracing;
+import brave.messaging.MessagingTracing;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.util.Properties;
+
+public final class ProducerService {
+
+  public static void main(String[] args) {
+    final MessagingTracing messagingTracing = MessagingTracingFactory.create("producer-service");
+    final KafkaTracing kafkaTracing = KafkaTracing.newBuilder(messagingTracing).build();
+
+    final Properties configs = new Properties();
+    configs.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:19092");
```

Comment:
Will this work in containarized apps? Maybe making it configurable is "the voice"?

Reply:
right. adding System property `kafka.bootstrap-servers`